environments/pull-environments.sh

```diff
@@ -1,11 +1,79 @@
 #!/bin/bash
 # using bin/bash for macOS and Linux compatibility
-set -e
-IFS='|'
```

Comment:
This is used for Amplify CLI usages, but is not needed in this file. Child script above handles it.